libs/ui/elements/src/lib/record-api-form/record-api-form.component.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { TestBed, ComponentFixture } from '@angular/core/testing'
 import { RecordApiFormComponent } from './record-api-form.component'
 import { DatasetServiceDistribution } from '@geonetwork-ui/common/domain/model/record'
-import { firstValueFrom } from 'rxjs'
+import { firstValueFrom, of } from 'rxjs'
```

Comment:
I think the of import is not used and can be removed.

Reply:
Indeed, thanks!